app/views/boats/index.html.erb

```diff
@@ -7,18 +7,23 @@
         <%= link_to "Add a new boat", new_boat_path, class: "btn btn-default btn-lg" %>
       </div>
     </div>
-
-    <div class="container">
+<div class="container">
       <div class="row">
         <% @boats.each do |boat| %>
-          <div class="col-md-4">
+          <div class="col-md-3">
```

Comment:
Ik vind 3 kolommen mooier dan 4, maar misschien komt dat ook doordat ik hem niet fullscreen bekijk. Moeten we het met de css nog over hebben denk ik.